cfgov/jinja2/v1/_includes/organisms/item-introduction.html

```diff
@@ -46,12 +46,12 @@
 {% import 'macros/time.html' as time %}
 <div class="o-item-introduction">
     <div class="h4">
-        {{ category_slug.render('info for consumers') }}
+        {{ category_slug.render(value.category) }}
     </div>
-    <h1>{{ value.title_heading }}</h1>
+    <h1>{{ value.heading }}</h1>
 
-    {% if value.paragraph_heading %}
-        <p class="short-desc">{{ value.paragraph_heading | safe }}</p>
+    {% if value.paragraph %}
+        <div class="short-desc">{{ value.paragraph.source | safe }}</div>
```

Comment:
documented, also just fyi we noticed that the wagtail rich text blocks would render outside of `<p>` tags, and also within their own `<div class="richtext">` blocks. so to avoid this we reference the `source` itself and wrapping them in `div` with relevant styling, i.e `"short-desc"`